uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java

```diff
@@ -722,21 +728,25 @@ public synchronized void handleLiveInstanceChange(boolean onlyCheckOffline) thro
                 break;
               }
               try {
-                Thread.sleep(10);
+                Thread.sleep(100);
```

Comment:
what's the reason to increase the sleep?

Reply:
Based on testing, the wait time if usually in the order of 100 ms, so lower the frequency to reduce the call to zookeeper.

Reply:
make sense, it's nice to have some comments.